client/src/topbar.jsx

```diff
@@ -7,6 +7,9 @@ import Typography from '@material-ui/core/Typography';
 import Button from '@material-ui/core/Button';
 import IconButton from '@material-ui/core/IconButton';
 import MenuIcon from '@material-ui/icons/Menu';
+import TextField from './textfield.jsx';
+import GraphSelector from './graphs/Scattered.jsx';
```

Comment:
How are you going to upload my version of GraphSelector here?
are we gonna get it from proxy and funnel it to here>?
just a thought